ex03.ex

```diff
@@ -101,7 +110,9 @@ defmodule Ex03 do
 
   """
 
-  def list_equal . . . "your code"
+  def list_equal([], []), do: true
```

Comment:
No deduction, but... this code allows

~~~
list_equal("wombat", 99)   #=> false
~~~